README.adoc

```diff
@@ -46,6 +46,32 @@ gem install rouge
 ----
 ====
 
+== Layout
+
+=== Empty Line after test case[[empty-line-after-test-case]]
+
+Add empty lines after each test methods. It make the code more readable.
```

Comment:
Isn't this already covered by the Ruby style guide?

Reply:
Did you mean this section -> https://github.com/rubocop-hq/ruby-style-guide#empty-lines-between-methods ??

Reply:
Ah, surely this is the responsibility of `Layout/EmptyLineBetweenDefs` 💦

Reply:
https://rubystyle.guide/#empty-lines-between-methods

Reply:
Got it. I should close this PR then 😄. Thanks.